### DISCUSSION TITLE
using present-icons from CDN runtime

### DISCUSSION THREAD

**Discussion by hoishing** [unocss/unocss#1747] [Q&A]
Hi~ I am trying to use preset-icons over CDN but no luck

```html
<script src="https://cdn.jsdelivr.net/npm/@unocss/runtime"></script>
<script src="https://cdn.jsdelivr.net/npm/@unocss/preset-icons"></script>
<script>
  window.__unocss = {
    presets: [
      presetIcons({
        cdn: "https://esm.sh/",
      }),
    ],
  };
</script>
```

I guess it's not correct to use `https://cdn.jsdelivr.net/npm/@unocss/preset-icons` this way, any suggestion?

**Comment by 729902288** [comment]
<script src="https://cdn.jsdelivr.net/npm/@unocss/runtime/mini.global.js"></script>
    <script src="https://cdn.jsdelivr.net/npm/@unocss/runtime/preset-icons.global.js"></script>
    <script>
      window.__unocss = {
        presets: [
          () =>
            window.__unocss_runtime.presets.presetIcons({
              scale: 1.2,
              cdn: "https://esm.sh/",
            }),
        ],
      };
    </script>
    <script src="https://cdn.jsdelivr.net/npm/@unocss/runtime/core.global.js"></script>

**Reply by alihammad99** [reply]
Do you know how to use transformers in CDN?
